sdcm/fill_db_data.py

```diff
@@ -416,9 +417,33 @@ class FillDatabaseData(ClusterTester):
                 [],
                 [['Samwise']]
             ],
-            'min_version': '2.0',
+            'min_version': '3.0',
+            'max_version': '',
+            'skip': ''},
+        # mv_with_eq_test: Check that you can query from materialized view
+        {
+            'create_tables': ["""
+                                CREATE TABLE mv_with_eq_test (
+                                  userid uuid PRIMARY KEY,
+                                  firstname text,
+                                  lastname text,
+                                  age int);""",
+                              'CREATE MATERIALIZED VIEW mv_byAge as SELECT * from mv_with_eq_test WHERE userid IS NOT NULL and age IS NOT NULL PRIMARY KEY (age, userid);'],
+            'truncates': ['TRUNCATE mv_with_eq_test'],
+            'inserts': [
+                "INSERT INTO mv_with_eq_test (userid, firstname, lastname, age) VALUES (550e8400-e29b-41d4-a716-446655440000, 'Frodo', 'Baggins', 32)",
+                "UPDATE mv_with_eq_test SET firstname = 'Samwise', lastname = 'Gamgee', age = 33 WHERE userid = f47ac10b-58cc-4372-a567-0e02b2c3d479",
+            ],
+            'queries': [
+                "SELECT firstname FROM mv_with_eq_test WHERE userid = 550e8400-e29b-41d4-a716-446655440000",
+                "SELECT firstname FROM mv_byAge WHERE userid = f47ac10b-58cc-4372-a567-0e02b2c3d479 AND age = 33"],
```

Comment:
How about add an invalid query?

- "SELECT firstname FROM mv_byAge WHERE userid = f47ac10b-58cc-4372-a567-0e02b2c3d479 AND age = 30"

And a query without the PK.

- "SELECT firstname FROM mv_byAge WHERE age = 33;"

Reply:
Does it have additional value for upgrade?
Anyway, the goal of this PR - validate, that using of SI after upgrade doesn't cause to the failure. 
If we need to revise data cases - it should be another task

Reply:
Ok, it's enough.